spec/controllers/api/users_controller_spec.rb

```diff
@@ -6,11 +6,12 @@
     let!(:user_2) { FactoryGirl.create(:user) }
     let(:result) { JSON.parse(response.body) }
     before do
+      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
```

Comment:
Line is too long. [139/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.